scripts/dom-helpers.js

```diff
@@ -64,44 +70,137 @@ export const removeEmptyTags = (block) => {
+ * @param {number} [timeout=3000] - The maximum time to wait for the element,
+ * in milliseconds. Defaults to 3000ms.
+ * @returns {Promise<Element>} A promise that resolves with the element when it is found,
+ * or rejects if the timeout is reached.
+ */
+export const waitForElement = (selector, timeout = 3000) => new Promise((resolve, reject) => {
+  const interval = 100;
+  let elapsedTime = 0;
+
+  const check = () => {
+    const element = document.querySelector(selector);
+    if (element) {
+      resolve(element);
+    } else {
+      elapsedTime += interval;
+      if (elapsedTime >= timeout) {
+        reject(new Error('Element not found: ', selector));
+      } else {
+        setTimeout(check, interval);
+      }
+    }
+  };
+
+  check();
+});
+
 /*
     More short hand functions can be added for very common DOM elements below.
     domEl function from above can be used for one off DOM element occurrences.
   */
-export function div(...items) { return domEl('div', ...items); }
-export function p(...items) { return domEl('p', ...items); }
-export function a(...items) { return domEl('a', ...items); }
-export function h1(...items) { return domEl('h1', ...items); }
-export function h2(...items) { return domEl('h2', ...items); }
-export function h3(...items) { return domEl('h3', ...items); }
-export function h4(...items) { return domEl('h4', ...items); }
-export function h5(...items) { return domEl('h5', ...items); }
-export function h6(...items) { return domEl('h6', ...items); }
-export function ul(...items) { return domEl('ul', ...items); }
-export function ol(...items) { return domEl('ol', ...items); }
-export function li(...items) { return domEl('li', ...items); }
-export function i(...items) { return domEl('i', ...items); }
-export function small(...items) { return domEl('small', ...items); }
-export function strong(...items) { return domEl('strong', ...items); }
-export function img(...items) { return domEl('img', ...items); }
-export function span(...items) { return domEl('span', ...items); }
-export function input(...items) { return domEl('input', ...items); }
-export function form(...items) { return domEl('form', ...items); }
-export function label(...items) { return domEl('label', ...items); }
-export function button(...items) { return domEl('button', ...items); }
-export function nav(...items) { return domEl('nav', ...items); }
-export function aside(...items) { return domEl('aside', ...items); }
-export function meta(...items) { return domEl('meta', ...items); }
-export function picture(...items) { return domEl('picture', ...items); }
-export function br() { return domEl('br'); }
-export function select(...items) { return domEl('select', ...items); }
-export function option(...items) { return domEl('option', ...items); }
-export function dl(...items) { return domEl('dl', ...items); }
-export function dt(...items) { return domEl('dt', ...items); }
-export function dd(...items) { return domEl('dd', ...items); }
```

Comment:
Please revert and put it back to this other format where it was much easier to read.

Reply:
reverted to previous formatting